Fatmylin/rails101
# Ch07 新增我的群組以及我的文章功能

### Feature
- Add: 我的群組，可以看到所有加入的群組
- Add: 我的文章，可以看到所有在群組裡的發言
- Add: 非群組成員無法進行留言或是編輯之前的留言
- Fix: 將管理員的標籤修改為紅色
### How to test
- run rails s
- Open localhost:3000
- try to create a discuss group
- try to join a group
- try to quit a group
- try to create post
- try to see what groups you are in
